docs/DeveloperGuide.md

```diff
@@ -168,6 +168,46 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 ## **Implementation**
 
 This section describes some noteworthy details on how certain features are implemented.
+### \[Insert Numbering\] Add feature
+The Edit feature is facilitated by `LogicManager`. The `AddCommandParser` parses the command arguments, and returns
```

Comment:
I think this should be The Add Feature is facilitated, just a small typo.

Reply:
Resolved, thanks for spotting it!